ibis/backends/snowflake/__init__.py

```diff
@@ -75,6 +78,11 @@ class SnowflakeExprTranslator(AlchemyExprTranslator):
 
 
 class SnowflakeCompiler(AlchemyCompiler):
+    @property
+    @staticmethod
```

Comment:
This isn't valid, `@property` operates on instance methods.

Reply:
Can you revert back to the way this was before?

`_NATIVE_ARROW` is defined at import time, not when the first query is run.